it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
 
 ## @name NS_ERROR_SMTP_PASSWORD_UNDEFINED
 12584=Si è verificato un errore durante l'invio della posta: impossibile ottenere la password per %S. Il messaggio non è stato inviato.
 
 ## @name NS_SMTP_PASSWORD_PROMPT_TITLE
-12585=SMTP Server Password Required
+12585=Password richiesta per server SMTP
 
 ## @name NS_ERROR_SMTP_TEMP_SIZE_EXCEEDED
-12586=La dimensione del messaggio che si cerca di inviare eccede temporaneamente il limite massimo del server. Il messaggio non è stato inviato; tentare di ridurre la dimensione del messaggio o aspettare un po' di tempo e riprovare. Il server ha risposto: %s.
+12586=Le dimensioni del messaggio che stai cercando di inviare superano un limite temporaneo del server. Il messaggio non è stato inviato; cerca di ridurre le dimensioni del messaggio o attendi e riprova. Il server ha risposto: %s.
 
 ## @name NS_ERROR_SMTP_PERM_SIZE_EXCEEDED_1
-12587=La dimensione del messaggio che si cerca di inviare eccede la dimensione massima globale (%d byte) del server. Il messaggio non è stato inviato; Ridurre la dimensione del messaggio e riprovare.
+12587=Le dimensioni del messaggio che stai cercando di inviare superano il limite globale (%d byte) del server. Il messaggio non è stato inviato; riduci le dimensioni del messaggio e riprova.
 
 ## @name NS_ERROR_SMTP_PERM_SIZE_EXCEEDED_2
-12588=La dimensione del messaggio che si cerca di inviare eccede la dimensione massima globale (%d byte) del server. Il messaggio non è stato inviato; Ridurre la dimensione del messaggio e riprovare. Il server ha risposto: %S.
+12588=La dimensione del messaggio che stai cercando di inviare superano il limite globale (%d byte) del server. Il messaggio non è stato inviato; riduci le dimensioni del messaggio e riprova. Il server ha risposto: %S.
 
 ## @name NS_ERROR_SMTP_SEND_FAILED_UNKNOWN_SERVER
-12589=Errore durante l'invio della posta: server SMTP %S sconosciuto. Il server potrebbe essere mal impostato. Controllare le impostazioni relative al server SMTP e riprovare.
+12589=Si è verificato un errore nell'invio della e-mail: server SMTP %S sconosciuto. Il server potrebbe non essere configurato correttamente. Verifica che le tue impostazioni del server SMTP siano corrette e riprova.
 
 ## @name NS_ERROR_SMTP_SEND_FAILED_REFUSED
-12590=Non è stato possibile inviare il messaggio in quanto impossibile connettersi al server SMTP %S. Il server potrebbe essere non disponibile o potrebbe rifiutare connessioni SMTP. Verificare che le impostazioni relative al server SMTP siano corrette e riprovare oppure contattare l'amministratore del server.
+12590=Non è stato possibile inviare il messaggio poiché la connessione al server SMTP %S non è riuscita. Il server potrebbe non essere disponibile o potrebbe rifiutare connessioni SMTP. Verifica che le impostazioni relative al server SMTP siano corrette e riprova oppure contattare l'amministratore del server.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12590=Non è stato possibile inviare il messaggio poiché la connessione al server SMTP %S non è riuscita. Il server potrebbe non essere disponibile o potrebbe rifiutare connessioni SMTP. Verifica che le impostazioni relative al server SMTP siano corrette e riprova oppure contatta l'amministratore del server.